draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -204,10 +207,10 @@ a type:
 
 ```
 {
-  "type": "IdentityCredential"
+  "type": "https://credentials.example.com/identity_credential"
 }
 ```
-For example, a type `IdentityCredential` can be defined such that at least the registered JWT claims `given_name`, `family_name`, `birthdate`, and `address` must appear in the Unsecured Payload. Additionally, the registered JWT claims `email` and `phone_number`, and the private claims `is_over_18`, `is_over_21`, and `is_over_65` may be used. The type might also indicate that `given_name`, `family_name`, `birthdate`, `address` can be selectively disclosable.
+For example, a type `https://credentials.example.com/identity_credential` can be associated with rules that define that at least the registered JWT claims `given_name`, `family_name`, `birthdate`, and `address` must appear in the Unsecured Payload. Additionally, the registered JWT claims `email` and `phone_number`, and the private claims `is_over_18`, `is_over_21`, and `is_over_65` may be used. The type might also indicate that `given_name`, `family_name`, `birthdate`, `address` can be selectively disclosable.
```

Comment:
I know it's not normative but the example elsewhere in the document uses all the same claims (and was the same type value) and has them all selectively disclosable. It'd be good to not give the impression of contradictory type definition even if it's not a real definition. 

```suggestion
For example, a type `https://credentials.example.com/identity_credential` can be associated with rules that define that at least the registered JWT claims `given_name`, `family_name`, `birthdate`, and `address` must appear in the Unsecured Payload. Additionally, the registered JWT claims `email` and `phone_number`, and the private claims `is_over_18`, `is_over_21`, and `is_over_65` may be used. The type might also indicate that any of the aforementioned claims can be selectively disclosable.
```

Reply:
Also, kinda related, the generated example(s) currently still have `type: IdentityCredential` (at [this yml](https://github.com/oauth-wg/oauth-sd-jwt-vc/blob/danielfett/fix-171/examples/01/specification.yml#L2) and [this other yml](https://github.com/oauth-wg/oauth-sd-jwt-vc/blob/danielfett/fix-171/examples/02/specification.yml#L4)) which of course isn't a Collision-Resistant Name.  Should probably update those to `https://credentials.example.com/identity_credential` to line of with the examples given in text and align with the requirement of being a Collision-Resistant Name.